File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
const twoSum = function (numbers, target) {
    let low = 0,
        high = numbers.length - 1;
    while (low < high) {
        const sum = numbers[low] + numbers[high];
        if (sum < target) low++;
        else if (sum > target) high--;
        else return [low + 1, high + 1];

    }
};

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        low,high=0,len(numbers)-1
        while(low<high):
            sum=numbers[low]+numbers[high]
            if sum==target:
                return [low+1,high+1]
            elif sum>target:
                high=high-1    
            elif sum<target:
                low=low+1